components/form/builder/src/index.js

```diff
@@ -18,21 +18,22 @@ import AtomSpinner, {AtomSpinnerTypes} from '@s-ui/react-atom-spinner'
 import {getUpdatedFormState} from './mapper/formState'
 
 const FormBuilder = ({
+  applyEmptyFields,
   json,
-  initFields,
-  onChange,
-  onFocus,
-  onBlur,
-  responseInterceptor,
-  requestInterceptor,
+  initFields = {},
+  onChange = () => {},
```

Comment:
Functions should not be empty (Code Smell –Critical)
https://rules.sonarsource.com/javascript/RSPEC-1186